extensions/amp-accordion/0.1/amp-accordion.js

```diff
@@ -68,7 +69,7 @@ class AmpAccordion extends AMP.BaseElement {
    */
   onHeaderClick_(event) {
     event.preventDefault();
-    const section = event.target.parentNode;
+    const section = closestByTag(event.target, 'section');
```

Comment:
From MDN i thought this would not work on safari, looks like it does, so proceeding with currentTarget.